platform/android/java/app/config.gradle

```diff
@@ -6,7 +6,8 @@ ext.versions = [
     buildTools         : '30.0.1',
     supportCoreUtils   : '1.0.0',
     kotlinVersion      : '1.4.10',
-    v4Support          : '1.0.0'
+    v4Support          : '1.0.0',
+    javaVersion        : 1.8
```

Comment:
Do you know what's the state of Java 11 compatibility in the Android SDK / Studio?

Our documentation still tells people to install specifically JDK 8 for `jarsigner`, as earlier and later versions seemed to be problematic, and I remember running into issues with newer Java when using `sdkmanager`.

But many Linux distros are finally leaving OpenJDK 1.8 behind and defaulting to JDK 11 LTS, so we'll need to support this somehow if we can.

Reply:
@akien-mga I'll look into it.